docs/written/trove/lists.scrbl

```diff
@@ -1623,13 +1623,104 @@ end
 }
 
 @list-method["member"]
+
+Returns true if the @pyret{elt} is located somewhere in the list.
+
+@examples{
+check:
+  [list: 'a', 'b', 'c'].member('a') is true
```

Comment:
I think that the way it's written is slightly better, since it illustrates an explicit usage of the method. Additionally, it follows the conventions that I'm seeing elsewhere in the documentation.